subjects/devops/calculator/README.md

```diff
@@ -0,0 +1,62 @@
+## Calculator
+
+### Instructions
+
+In this exercise you will make a script `calculator.sh` that will take 3 arguments, calculate the result and print it on the standard output.
+
+- The first argument and the third argument will be numbers.
+- The second argument will be the operator.
+
+Each operator should have its own function named as follow:
+- `+`: `do_add()`.
+- `-`: `do_sub()`.
+- `*`: `do_mult()`.
+- `/`: `do_divide()`.
+
+Each function will receive two arguments, the left number and the right number, and return the result of the operation.
+
+The functions assume the input is valid, so the input should be checked before to call the functions.
+
+> The functions will also be tested individually, so it is important to name each function exactly as above, the behavior of the functions have to match the exercise instructions.
+
+### Usage
+
+```console
+$ ./calculator.sh 20 "*" 3
+60
+$ ./calculator.sh 20 / 20
+1
+$ ./calculator.sh -1 - 10
+-11
+$
+```
+
+### Error handling
+
+All errors will print a specific message on **stderr** (ending with a newline) and returns a specific non-zero value:
+- Wrong number of arguments: `"Error: expect 3 arguments"`, returns `1`.
+- Division by 0: `"Error: division by 0"`, exit with `2`.
+- Invalid operator: `"Error: invalid operator"`, exit with `3`.
+- Invalid number(s): `"Error: invalid number"`, exit with `4`.
+
+> Negative numbers are also a valid input.
+
+### Hints
```

Comment:
show an example of the "case" condition here, as this exercise is supposed to introduce case to the students.
also if you can maybe we should add a verification in the test to check if they use case or not.
basically make case a mandatory condition for this exercise, what do you think?

Reply:
- I've explicitly added the usage of `case` as a requirement.
- Also now there is an example of `case` usage in the `Hints` paragraph.

The usage of case is now strongly encouraged and it looks mandatory to the student, but in terms of automated tests I don't think it is possible to check that the student actually uses `case`.
If you have an idea on how to add this automated verification in tests I'll be glad to implement it, that could be very interesting to do actually.

Reply:
Im not sure as well if it is possible, maybe with a grep like we use to check if they dont use echo
```
FILENAME="student/calculator.sh"

if [[ $(cat $FILENAME | grep case | wc -l) -ne 1 ]]; then
    echo "case is mandatory!"
fi
```
but i guess if it is not we can just encouraged on the readme

Reply:
Thanks for the example, I've modified it slightly and added it so we have a test (even if a bit partial) for usage of case statement.